frontend/src/store/app/actions.ts

```diff
@@ -15,14 +15,10 @@ export function setLocale(locale: string): AppAction {
   return { type: SET_LOCALE, locale };
 }
 
-export function lostBackendConnection(): AppAction {
-  return { type: BACKEND_CONNECTION_DOWN };
+export function lostBackendConnection<T extends PBMessage>(updates: Partial<T>): AppAction {
```

Comment:
Is there a way we can just define a separate type for this action?

Reply:
If we can do it in the reducer then i guess we don't need this!